app/cdap/components/DataPrep/Directives/ChangeDataType/DecimalOptions.tsx

```diff
@@ -0,0 +1,293 @@
+                    >
+                      <HelpIcon />
+                    </IconButton>
+                  </Tooltip>
+                </InputAdornment>
+              }
+            />
+            <FormHelperText>{T.translate(`${PREFIX}.scaleHelperText`)}</FormHelperText>
+          </FormControl>
+          <Tooltip
+            arrow
+            placement="top"
+            disableFocusListener={!isRoundingDisabled}
+            disableHoverListener={!isRoundingDisabled}
+            disableTouchListener={!isRoundingDisabled}
+            title={T.translate(`${PREFIX}.roundingDisabledTooltip`).toString()}
+            classes={{ tooltip: classes.customTooltip, arrow: classes.customTooltipArrow }}
+          >
+            <FormControl variant="outlined" disabled={isRoundingDisabled}>
+              <InputLabel id={ROUNDING_LABEL_ID} htmlFor={ROUNDING_INPUT_ID}>
+                {T.translate(`${PREFIX}.roundingLabel`)}
+              </InputLabel>
+              <Select
+                labelId={ROUNDING_LABEL_ID}
+                id={ROUNDING_INPUT_ID}
+                value={rounding}
+                onChange={handleRoundingChange}
+                label={T.translate(`${PREFIX}.roundingLabel`)}
+                endAdornment={
+                  <InputAdornment position="end">
+                    <Tooltip
+                      arrow
+                      enterDelay={500}
+                      title={T.translate(`${PREFIX}.roundingTooltip`).toString()}
+                      classes={{
+                        tooltip: classes.customTooltip,
+                        arrow: classes.customTooltipArrow,
+                      }}
+                    >
+                      <IconButton
+                        aria-label={T.translate(`${PREFIX}.roundingHelpButton`).toString()}
+                        edge="end"
+                      >
+                        <HelpIcon />
+                      </IconButton>
+                    </Tooltip>
+                  </InputAdornment>
+                }
+              >
+                {ROUNDING_MODES.map((mode: IRoundingMode) => (
+                  <MenuItem value={mode.value} key={mode.value}>
+                    <Tooltip
+                      arrow
+                      enterDelay={500}
+                      title={mode.description}
+                      classes={{
+                        tooltip: classes.customTooltip,
+                        arrow: classes.customTooltipArrow,
+                      }}
+                    >
+                      <span>{mode.text}</span>
+                    </Tooltip>
+                  </MenuItem>
+                ))}
+              </Select>
+              <FormHelperText>{T.translate(`${PREFIX}.roundingHelperText`)}</FormHelperText>
+            </FormControl>
+          </Tooltip>
+          <ButtonsContainer>
+            <Button variant="contained" color="primary" disableElevation onClick={applyDirective}>
```

Comment:
nit: Can't we use any of the button components present in shared components here?

Reply:
yes we can. But we are going ahead with using MUI components as much as possible, for uniformity.

Reply:
BTW we have these button components: https://github.com/cdapio/cdap-ui/tree/develop/app/cdap/components/shared/Buttons based on Mui for consistency.